packages/shared/lib/sdk/sync.ts

```diff
@@ -801,10 +801,6 @@ export class NangoAction {
     }
 
     public async getEnvironmentVariables(): Promise<EnvironmentVariable[] | null> {
-        if (!this.environmentId) {
```

Comment:
I don't know enough to know for sure, but would it be better for us to return an empty array if there's no environmentId set?

Reply:
Well, for a getConnection call environmentId is never going to be set https://docs.nango.dev/reference/api/connection/get -- I believe at some point we did get back an environmentId. In order to get an environment there is no need to have an environment id present either https://docs.nango.dev/reference/api/sync/environment-variables

Reply:
the environment is deducted from the secretKey that is used to authenticate the call to `getEnvironmentVariables`. As you can see below `this.environmentId` is not even passed to `getEnvironmentVariables`